analytics.html

```diff
@@ -1,65 +1,90 @@
 <html>
-    <head>
-        <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Crimson+Text" />
-        <link rel="stylesheet" type="text/css" href="stylesheet.css">
-        <script src="https://use.fontawesome.com/432212f1dd.js"></script>
-        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
-        <script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.17.1/moment.min.js"></script>
-    </head>
-    <body>
-        <div class="title-bar box">
-            <div id="date">Tuesday, February 14, 2017</div>
-            <div id="title">Ubyssey Analytics</div>
-            <div id="time"><span>3:04pm</span></div>
-        </div>
+  <head>
+    <link rel="stylesheet" type="text/css" href="stylesheet.css">
+    <script type="text/javascript" src="https://use.fontawesome.com/432212f1dd.js"></script>
+    <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
+    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.17.1/moment.min.js"></script>
+    <script type="text/javascript" src="http://d3js.org/d3.v3.js"></script>
```

Comment:
`text/javascript` can easily be excluded if desired, since browsers assume it anyways